index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+    <html>
+        <head>
+            <link rel="stylesheet" href="styles/reset.css"/>
```

Comment:
Make sure that you are including the following meta tag so that your site is responsive (if you open up your console and click the mobile view, you will see that it does not actually reflect the mobile view):

`<meta name="viewport" content="device-width, initial-scale=1">`